4703 - Don't require WixBalExtension when using WixNetFxExtension

rseanhall opened the pull request.

robmen commented on src/chm/documents/customactions/wixnetfxextension.html.md:
```diff
@@ -7,122 +7,6 @@ after: using_standard_customactions
 
 The [WixNetfxExtension](~/xsd/netfx/index.html) includes a set of custom actions to compile native images using Ngen.exe. For an example, see [How To: NGen managed assemblies during installation](~/howtos/files_and_registry/ngen_managed_assemblies.html).
 
-## PackageGroups
```
@barnson good idea to move all these?

barnson replied:
I think reference material like this should stay with its "owning" topic. How-tos are less discoverable.

rseanhall replied:
I think I should have linked to the how-to from here.  It doesn't make sense to me for bundle information like this to be under the `customactions` folder, I don't think that's discoverable either.  Maybe create a NetFxExtension page under the `bundles` folder for this information?

barnson replied:
Yeah, that's a bad name. The content is about the extensions and everything they contain. Probably an artifact of people finding a place that mostly fits. :)

robmen commented on src/ext/BalExtension/wixlib/MbaPrereqs.wxs:
```diff
@@ -0,0 +1,168 @@
+<?xml version='1.0' encoding='utf-8'?>
```
Okay, but should we break this out into separate files? I thought @barnson did that when the Nefx.wxs got too crazy with everything in one file. Thoughts?

rseanhall replied:
I'm neutral on this, I can break them out.

rseanhall committed:
WIXFEAT: 4703 - Make NetFx extension not require the Bal extension.

Move the PrereqPackage information out of the NetFx extension and into the Bal extension so that the NetFx extension doesn't require the Bal extension.
